backend/controller/controller.go

```diff
@@ -285,14 +285,14 @@ func (s *Service) Status(ctx context.Context, req *connect.Request[ftlv1.StatusR
 	deployments, err := slices.MapErr(status.Deployments, func(d dal.Deployment) (*ftlv1.StatusResponse_Deployment, error) {
 		labels, err := structpb.NewStruct(d.Labels)
 		if err != nil {
-			return nil, fmt.Errorf("could not marshal attributes for deployment %s: %w", d.Name, err)
+			return nil, fmt.Errorf("could not marshal attributes for deployment %s: %w", d.Key.String(), err)
```

Comment:
FYI you don't need to use `.String()` with `%s` - the formatter will do this automatically.

Reply:
Oh neat